File: tools-manage/invocationlab-admin/src/main/java/org/javamaster/invocationlab/admin/model/erd/UsersResVo.java
package org.javamaster.invocationlab.admin.model.erd;

import lombok.Data;

import java.io.Serializable;
import java.util.List;

/**
 * @author yudong
 */
@Data
public class UsersResVo implements Serializable {
    private Integer total;
    private Integer size;
    private Integer current;
    private boolean searchCount;
    private Integer pages;
    private List<UsersVo> records;
    private List<Object> orders;

}

File: tools-manage/invocationlab-admin-21/src/main/kotlin/org/javamaster/invocationlab/admin/model/erd/UsersResVo.kt
package org.javamaster.invocationlab.admin.model.erd

import org.javamaster.invocationlab.admin.annos.AllOpen

import java.io.Serializable

/**
 * @author yudong
 */

@AllOpen
class UsersResVo : Serializable {
    var total: Int? = null
    var size: Int? = null
    var current: Int? = null
    var searchCount = false
    var pages: Int? = null
    var records: List<UsersVo>? = null
    var orders: List<Any>? = null
}
